src/core/Form/SearchInput/__snapshots__/SearchInput.test.tsx.snap

```diff
@@ -244,6 +244,22 @@ exports[`calling render with the same component on the same container does not r
   box-shadow: 0 1px 2px 0 rgba(0,53,122,0.1) inset;
 }
 
+.c2.fi-text-input_with-icon .fi-text-input_container {
```

Comment:
If it has no effect for other use, I'd prefer to set the container always `position: relative`